src/views/AllContacts.vue

```diff
@@ -1,5 +1,121 @@
 <template>
-  <div class="about">
-    <h1>This is a list page</h1>
-  </div>
+  <v-container fluid id="containerAll">
+    <v-text-field
+      v-model="search"
+      append-icon="mdi-magnify"
+      label="Search for a contact"
+      hide-details
+    ></v-text-field>
+    
+    <Pagination
+      v-if="tableData"
+      v-model="pagination"
+      :totalRecords="tableData.length"
+      :perPageOptions="perPageOptions"
+    />
+
+    //This table it's working only with search bar
+    <v-data-table
+      :headers="config"
+      :items="tableData"
+      :search="search"
+      :items-per-page = "perPageOptions"
+      :pagination="pagination"
+    ></v-data-table>
+
+      <v-divider></v-divider>
+      <v-divider></v-divider>
+      <v-divider></v-divider>
+      
+    //This table it's working only with pagination
+    <Table
+      v-if="tableData"
+      :theData="computedTableData"
+      :config="config"
+      :perPage="perPageOptions"
+      :search="search"
+    />
+  </v-container>
 </template>
+
+<script>
+import Table from '../components/Table'
+import Pagination from '../components/Pagination'
+import mockData from '../store/mock-data.js'
+const perPageOptions = [6]
+
+export default {
+  components: {
+    Table,
+    Pagination
+  },
+  data: function () {
+    return {
+      perPageOptions,
+      page: 1,
+      perPage: perPageOptions[0],
+      tableData: mockData,
+      pagination: { page: 1, perPage: perPageOptions[0] },
+      search: '',
+      config: [
+        {
+          value: 'email',
+          text: 'Email'
+        },
+        {
+          text: 'Source'
+        },
+        {
+          value: 'familyName',
+          text: 'First Name'
+        },
+        {
+          value: 'givenName',
+          text: 'Last Name'
+        },
+        {
+          value: 'applicationStatus',
+          text: 'App Status'
+        },
+        {
+          text: 'Profile Status'
+        },
+        {
+          value: 'value',
+          text: 'Value'
+        },
+        {
+          value: 'mortgageRenewalDate',
+          text: 'Renewal Date',
```

Comment:
For this one maybe will be ok to format the date in local date and the table to get the formatted date. Also, for value column from line 85, the table to be able to get the` "value" + "currency"`.